xsd/NeTEx_publication.xsd

```diff
@@ -16,28 +16,48 @@
 				<Contributor>CEN TC278 WG3 SG9 Team.</Contributor>
```

Comment:
Only do the very relevant change in NeTEx_publication.xsd. Otherwise we are in trouble, because the PR is that old.